meta-xt-prod-devel-rcar-control/recipes-guests/domu/files/domu-vdevices-virtio.cfg

```diff
@@ -8,6 +8,7 @@ virtio=[
 'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:03.0, grant_usage=1, backend_type=qemu',
 'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:04.0, grant_usage=1, backend_type=qemu',
 [VIRTIO_DRIVE_CONFIGURATION]
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:06.0, grant_usage=1, backend_type=qemu'
```

Comment:
Maybe it is better to put this before `[VIRTIO_DRIVE_CONFIGURATION]`? This will save you change to domu-set-root. Or you can just add comma after `[VIRTIO_DRIVE_CONFIGURATION]`